### PR TITLE
Adds lizard wine to the game.

### PR DESCRIPTION
**Pull request by harmonyn**
fucking lizards

Lizard wine is a real drink, Asian in origin: 

http://cdn-image.foodandwine.com/sites/default/files/images-sys-201010-wines-lizard-ss.jpg 

http://www.euvs.org/img/spirits/full/1547.jpg

It only makes sense that Space Station 13 would adopt this recipe to deal with excess lizards.

blah blah fix here https://github.com/tgstation/tgstation/pull/18395

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -288,6 +288,12 @@
 /obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe/premium/redact()
 	return
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/lizardwine
+	name = "Bottle of lizard wine"
+	desc = "An alcoholic beverage from Space China, made by infusing lizard tails in ethanol. Inexplicably popular among command staff."
```
Comment (Cheridan) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: uh this is most certainly not canon

**Comment by optimumtact** [comment]
Actually it doesn't make much sense when you consider the lore is most humans are racist but centcomm allows lizards because they're cheap, why would they turn cheap workers into wine?

**Comment by bgobandit** [comment]
I literally just fixed two issues today

**Comment by optimumtact** [comment]
you need to link to one in your PR OP as I dont' have time to manually manage people's merge tokens

**Comment by bgobandit** [comment]
also it's just their tails it's not the whole lizard

**Comment by Zilenan91** [comment]
They regrow their tails anyways. "unfortunate accidents" sure are terrible huh

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm (hunk 1 above)
Comment (bgobandit) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: take it to #lorebus

Comment (Iamgoofball) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: i think space china is legit

**Comment by Anonmare** [comment]
I thought we all had enough of lizard whine?

**Pull request merged**